src/frontend/Sider/Behandling/Felles/BekreftEndretDatoetFørTidligereVedtak/BekreftEndringPåPeriodeSomPåvirkerTidligereVedtakModal.tsx

```diff
@@ -18,6 +19,12 @@ export const BekreftEndringPåPeriodeSomPåvirkerTidligereVedtakModal = ({
     laster: boolean;
 }) => {
     const { sluttDatoForrigeVedtak } = useBehandling();
+    if (visBekreftModal) {
+        sendHendelseTilUmami(
+            UmamiHendelse.VIS_BEKREFT_ENDRING_SOM_PÅVIRKER_TIDLIGERE_VEDTAK_MODAL,
+            { skalVises: visBekreftModal }
```

Comment:
Det kan sikkert være greit å forholde oss til [NAV-taksonomien](https://startumami.ansatt.nav.no/taksonomi) der det gir mening

Det ville her blitt noe ala 
```suggestion
        sendHendelseTilUmami(
            'modal åpnet',
            {
                modalId: 'bekreft-endring-av-tidligere-periode',
                tittel: 'Endring av periode som kanskje påvirker tidligere vedtak'
            }
```